notebooks/olmocr-pdf-vlm/olmocr-pdf-vlm.ipynb

```diff
@@ -0,0 +1,624 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #6.    display(Markdown(json.loads(text_output[0].replace("&lt;|im_end|&gt;", ""))["natural_text"]))
</pre><p>in model card sample, authors use skip_special_tokens=True. Your code above uses False. Is there some specific needs for it?</p><p> </p><p>I believe "&lt;|im_end|&gt;" token will not appear if use use skip_special_tokens=True in decoding</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/openvino_notebooks/pull/2842/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>

Reply:
<p>Fixed</p>